aws/README.md

```diff
@@ -100,3 +101,31 @@ Assumes the following are installed through choco:
 ```
 tag_instance.ps1
 ```
+
+## update_other_tag.py
+
+Update the value of CostCenterOther for a specified EC2 instance, including related resources in the same cloudformation stack and/or service catalog product.
+
+### Dependencies
+
+The following python packages are required:
+
+* boto3
+* requests
+
+### Usage
+
+#### Options
+
+* -r --resource_id: Target resource to tag, e.g. EC2 instance name
+* -t --tag_value:   New value for CostCenterOther tag
+
+#### Environment Variables
+
+* AWS_PROFILE:  Set AWS account based on ~/.aws/config
+* MIPS_API_URL: Overwrite default mips-api URL
+
+#### Example
+```
+AWS_PROFILE=org-sagebase-scipooldev update_other_tag.py -r i-061fee3df7b496cd5 -t "No Program / 000000"
```

Comment:
this example doesn't set the `MIPS_API_URL` because it  seems like there's a default URL?  It would be good to document the default URL either here or in the CLI help output.

Reply:
done